config.go

```diff
@@ -24,7 +24,13 @@ type Config struct {
 	// TableName is name of table to scan
 	TableName string
 
-	// TotalSegments determines amount of concurrency to scan table with
+	// SegmentIndex determines where to start indexing from
+	SegmentIndex int
+
+	// SegmentSize deterines how big a segment is
+	SegmentSize int
```

Comment:
how about `SegmentCount` and `SegmentOffset` and then could remove `TotalSegments`? could just update the README `TotalSegments->SegmentCount`?